src/test/kotlin/CheckersModelTest.kt

```diff
@@ -1,11 +1,116 @@
+import org.junit.jupiter.api.DynamicTest
```

Comment:
Be free to split this file into several ones, I think that the time has come

Reply:
Fixed